project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -132,19 +159,27 @@ public static Document createDocument(String name, String language, String hash,
       Entity docEntity = new Entity("Document");
       ArrayList<Long> editorIDs = new ArrayList<Long>();
       ArrayList<Long> viewerIDs = new ArrayList<Long>();
+      ArrayList<Long> commentIDs = new ArrayList<Long>();
       long folderID = Folder.DEFAULT_FOLDER_ID;
       
+      if(language.equals("C++")) {
```

Comment:
we might want to refactor our codebase so we have a txt with the mappings and take care of this on the frontend instead

Reply:
sounds good